### ISSUE TITLE
Issue with Realm

### ISSUE THREAD

**Issue by kimdv** [danielgindi/Charts#2195]
I try to use the Realm part of Chart, but I get this error when I tries to install the pods: 

```
Update all pods
Updating local specs repositories
Analyzing dependencies
[!] Unable to satisfy the following requirements:

- `RealmSwift (~> 2.4.3)` required by `Podfile`
- `RealmSwift (~> 1.1)` required by `Charts/Realm (3.0.1)`
```

My pod file

```
# Uncomment the next line to define a global platform for your project
# platform :ios, '10.0'

target 'MyProjectName' do
  # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
  use_frameworks!

  # Pods for MyProjectName


  pod 'Realm', '~> 2.4.3'
  pod 'RealmSwift', '~> 2.4.3'

  pod 'Charts/Realm', '~> 3.0.1'
end
```

**Comment by petester42** [comment]
That's because 'Charts/Realm' uses 1.1. This should be updated. If you don't need realm 2 features simply remove the 'Realm' and 'RealmSwift' from your podifle and it should work

**Comment by kimdv** [comment]
I should be updated then, because I uses a lot of the new features.

**Comment by petester42** [comment]
You can always send a pull request updating realm for charts. I'll gladly review it!

**Comment by kimdv** [comment]
#2199

**Comment by petester42** [comment]
closed since it was addressed by #2199
